src/lib/app-helper.js

```diff
@@ -567,6 +570,30 @@ function getObjectValue (obj, key) {
   return keys.filter(o => o.trim()).reduce((o, i) => o && getObjectProp(o, i), obj)
 }
 
+/**
+ * Function to run `aio config get ims.contexts.cli.access_token.token` command and retrieve the token
+ * This function is used to check if the access token exists in the CLI context or not
+ * @returns {Promise<boolean>} Resolves to the access token if it exists, else false
+ */
+const checkifAccessTokenExists = async () => {
+  try {
+    const { stdout, stderr } = await execAsync('aio config get ims.contexts.cli.access_token.token')
```

Comment:
use findCommand to double check command existence, see https://github.com/adobe/aio-cli-plugin-app/blob/9b36d06c0bc0614f0744aba87324cdf4c2b7059a/src/commands/app/pack.js#L165

Probably can use execa like in the above link.

Reply:
Is using execa the new way to run other commands or should we use `.load()` and `.run()` like here

https://github.com/adobe/aio-cli-plugin-app/blob/251f7bb62424d52d9eef307a161d28b3ddcc9d9f/src/commands/app/run.js#L154-L158

Reply:
@MichaelGoberling actually those interfaces are not documented and we have to read the code to see how to do it. Perhaps they may go away? Unlikely, but I didn't even know about this interface. Even `findCommand` is not documented anymore, and `runCommand` is there instead: https://oclif.io/docs/config (not that they document how to use it!).

Not sure how that interface works if we need stdout/stderr.

Edit: I just saw they just have this guide: https://oclif.io/docs/running_programmatically/ which uses runCommand